src/commands.rs

```diff
@@ -0,0 +1,239 @@
+use crate::install::populate_data_dir;
+use crate::sessions::kill_session as kill_session_impl;
+use crate::sessions::{
+    assert_session, assert_session_ne, get_active_session, get_sessions,
+    get_sessions_sorted_by_creation_date, print_sessions, print_sessions_with_index,
+    session_exists, ActiveSession,
+};
+use dialoguer::Confirm;
+use std::path::PathBuf;
+use std::process;
+use zellij_client::start_client as start_client_impl;
+use zellij_client::{os_input_output::get_client_os_input, ClientInfo};
+use zellij_server::os_input_output::get_server_os_input;
+use zellij_server::start_server as start_server_impl;
+use zellij_utils::input::options::Options;
+use zellij_utils::nix;
+use zellij_utils::{
+    cli::{CliArgs, Command, SessionCommand, Sessions},
+    setup::{get_default_data_dir, Setup},
+};
+
+pub(crate) use crate::sessions::list_sessions;
+
+fn kill_all_sessions_impl(sessions: Vec<String>) {
+    for session in sessions.iter() {
+        kill_session_impl(session);
+    }
+    process::exit(0)
+}
+
+pub(crate) fn kill_all_sessions(yes: bool) {
+    match get_sessions() {
+        Ok(sessions) => {
+            if sessions.is_empty() {
+                println!("No active zellij sessions found.");
+                process::exit(1);
+            } else if yes {
+                kill_all_sessions_impl(sessions);
+            } else {
+                println!("WARNING: this action will kill all sessions.");
+                if Confirm::new()
+                    .with_prompt("Do you want to continue?")
+                    .interact()
+                    .unwrap()
+                {
+                    kill_all_sessions_impl(sessions);
+                } else {
+                    println!("Abort.");
+                    process::exit(1);
+                }
+            }
+        }
+        Err(e) => {
+            eprintln!("Error occurred: {:?}", e);
+            process::exit(1);
+        }
+    }
+}
+
+pub(crate) fn kill_session(target_session: &Option<String>) {
```

Comment:
@jaeheonji 
Thanks for your suggestion!

How do you think the naming convention won't be unified if renaming `kill_session` to `execute_kill_session ` by seeing it from the main function? If you would do so, all such as `start_server` should be renamed with `execute_start_server` respectively.

Reply:
The first idea was to prevent (more clarify) the name of
`kill_session` from being used in two places(`sessions.rs` and `command.rs`). But as you told me, it looks right to change start_server for unity. I want to respect your opinion here. What do you think it's better to do?

Reply:
@jaeheonji 
I personally mainly do not care about duplication of function names. I would think that's why there is a module system and visibility. We can rename it with `as` or import an upper namespace instead.

However, this is just my personal view. I think we need code owners' opinions. How do you think about this problem, @a-kenji ?

Reply:
@ken-matsui 
I currently prefer the simpler names here, especially since in the main function the commands are already called with:
```
commands::kill_session(target_session);
```

If it turns out to be confusing we can still change it in the future.
Thanks for the insight here @jaeheonji.